system-tests/azure-tests/src/test/java/org/eclipse/dataspaceconnector/system/tests/local/BlobTransferIntegrationTest.java

```diff
@@ -0,0 +1,238 @@
+                    "web.http.data.port", String.valueOf(PROVIDER_MANAGEMENT_PORT),
+                    "web.http.data.path", PROVIDER_MANAGEMENT_PATH,
+                    "web.http.ids.port", String.valueOf(PROVIDER_IDS_API_PORT),
+                    "web.http.ids.path", IDS_PATH,
+                    "ids.webhook.address", PROVIDER_IDS_API));
+
+
+    @BeforeAll
+    static void beforeAll() {
+        setUpMockVault(consumer, CONSUMER_VAULT);
+        setUpMockVault(provider, PROVIDER_VAULT);
+    }
+
+    private static void setUpMockVault(EdcRuntimeExtension consumer, Vault vault) {
+        consumer.registerServiceMock(Vault.class, vault);
+        consumer.registerServiceMock(PrivateKeyResolver.class, new NullVaultExtension().getPrivateKeyResolver());
+        consumer.registerServiceMock(CertificateResolver.class, new NullVaultExtension().getCertificateResolver());
+    }
+
+    @Test
+    public void transferBlob_success() {
+        // Arrange
+        // Upload a blob with test data on provider blob container (in account1).
+        var blobContent = "BlobTransferIntegrationTest-" + UUID.randomUUID();
+        createContainer(blobServiceClient1, PROVIDER_CONTAINER_NAME);
+        blobServiceClient1.getBlobContainerClient(PROVIDER_CONTAINER_NAME)
+                .getBlobClient(PROVIDER_ASSET_FILE)
+                .upload(BinaryData.fromString(blobContent));
+
+        // Seed data to provider
+        createAsset();
+        var policyId = createPolicy();
+        createContractDefinition(policyId);
+
+        // Write Key to vault
+        CONSUMER_VAULT.storeSecret(format("%s-key1", account2Name), account2Key);
+        PROVIDER_VAULT.storeSecret(format("%s-key1", account1Name), account1Key);
+
+        // Act
+        System.setProperty(ACCOUNT_NAME_PROPERTY, account2Name);
+        runGatling(BlobTransferLocalSimulation.class, TransferSimulationUtils.DESCRIPTION);
+
+        // Assert
+        var container = getProvisionedContainerName();
+        var destinationBlob = blobServiceClient2.getBlobContainerClient(container)
+                .getBlobClient(PROVIDER_ASSET_FILE);
+        assertThat(destinationBlob.exists())
+                .withFailMessage("Destination blob %s not created", destinationBlob.getBlobUrl())
+                .isTrue();
+        var actualBlobContent = destinationBlob.downloadContent().toString();
+        assertThat(actualBlobContent)
+                .withFailMessage("Transferred file contents are not same as the source file")
+                .isEqualTo(blobContent);
+    }
+
+    private String getProvisionedContainerName() {
+        return given()
+                .baseUri(CONSUMER_CONNECTOR_MANAGEMENT_URL + CONSUMER_MANAGEMENT_PATH)
+                .when()
+                .get(TRANSFER_PROCESSES_PATH)
+                .then()
+                .statusCode(200)
+                .extract().body()
+                .jsonPath().getString("[0].dataDestination.container");
```

Comment:
otherwise, test fails for me.

```suggestion
                .jsonPath().getString("[0].dataDestination.properties.container");
```

Json response of /transferprocess e.g.

```
	"dataDestination": {
		"properties": {
			"container": "a6ed5b83-20d6-4869-8dd1-1a6f34218eee",
```